sdk/servicebus/service-bus/src/log.ts

```diff
@@ -52,9 +52,9 @@ export const managementClientLogger = createServiceBusLogger("service-bus:manage
  * @param error - Error containing a stack trace.
  * @internal
  */
-export function logErrorStackTrace(_logger: AzureLogger, error: any): void {
-  if (error && error.stack) {
-    _logger.verbose(error.stack);
+export function logErrorStackTrace(_logger: AzureLogger, error: unknown): void {
+  if (error && Object.hasOwnProperty.call(error, "stack")) {
+    _logger.verbose((error as any).stack);
   }
```

Comment:
So, "any" is bad but "unknown" is fine?

Reply:
unknown is better than any because it prevents you from making assumptions on what properties/methods exist on the object. But here, I was wrestling with my approach.  Since we do an if check on the existence of `stack` on the error object, the spirit of the linter rule is actually followed which is that we are not making assumptions.

A better approach might be to do the below after using unknown in the function signature and use a separate `isErrorWithStack` like how we have `isServiceBusMessage()`

Reply:
Updated using the approach @chradek has for Event Hubs. See https://github.com/Azure/azure-sdk-for-js/pull/13935/commits/0c720a0c95fc1ebe1a59196689ae85300e1c7f67